pkg/rules/rules.go

```diff
@@ -5,6 +5,8 @@ package rules
 
 import (
 	"context"
+	"github.com/thanos-io/thanos/pkg/errutil"
```

Comment:
can you group this with the other thanos-io imports?

Reply:
do you mean 
```go
import (
		"github.com/thanos-io/thanos/pkg/errutil"
)
```

Reply:
Ah, no i meant to put it on line 20 roughly where rules/rulespb and tracing are imported

Reply:
of course